docs/generic-skills.md

```diff
@@ -0,0 +1,275 @@
+The AEA generic skills demonstrate an interaction between two AEAs.
+
+* The provider of the data.
+* The buyer of the data.
+
+## Preparation instructions
+ 
+### Dependencies
+
+Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start.
+
+##Discussion
+
+The scope of the specific demo is to demonstrate how to create an easy configurable AEA. The seller AEA
+will read data from `skill.yaml` file and will deliver to the buyer upon payment. To keep the demo simple we avoided the usage of a database or the usage of a hardware sensor since this would increase the complexity.
+As a result, the AEA can provide data that are listed in the `skill.yaml` file.
+This demo does not utilize a smart contract. We interact with a ledger only to complete a transaction.
+
+Moreover, this example provides a way to write your code to collect data if you want. 
+You can modify the `has_sensor` variable in `skill.yaml` file of the generic_seller skill to True. Then you have to implement the method `collect_data(self)` inside the strategy.py file. 
```

Comment:
I would relabel this variable as `has_data_source` and then `collect_from_data_source`